pytest.ini

```diff
@@ -1,9 +1,10 @@
 [pytest]
+addopts = -n8
 markers =
     serial: mark a test that requires more resources to run that are thus only suitable for serial run.
     remote_required: mark a test that requires internet access.
     gpu: mark a test that requires GPU.
     integration: mark an integration test
     skip_master: mark a test that is temporarily skipped for mxnet master validation.
 
-timeout = 30
+timeout = 300
```

Comment:
Why this?

Reply:
Some tests timed out before. Now, with using more workers, individual tests might even be slower.